modules/javafx.graphics/src/main/java/com/sun/javafx/sg/prism/NGSubScene.java

```diff
@@ -206,6 +206,10 @@ protected void renderContent(Graphics g) {
                                               msaa);
             }
             Graphics rttGraphics = rtt.createGraphics();
+            // The pixel scale factors must be set to the rttGraphics, otherwise the position
+            // of the lights will not be scaled correctly on retina displays
+            // See https://bugs.openjdk.java.net/browse/JDK-8255015
```

Comment:
We typically don't put pointers to bug IDs in the code (we used to, and still sometimes do in some special cases, but here it isn't needed).